cmd/controller-manager/main.go

```diff
@@ -44,10 +56,46 @@ type ProvideFunc func(manager.Manager) (controller.Controller, error)
 
 func main() {
 	flag.Parse()
```

Comment:
Not an issue w/ your PR, but we should consider renaming these binaries slightly; 'controller' runs the bus controllers and 'controller-manager' runs the flow, feed, etc controllers.

Reply:
I'll create an issue for this.

Reply:
I have some code that combines these into a single deployment, but haven't PR'd it: https://github.com/knative/eventing/compare/master...grantr:combine-controllers

Reply:
#313